estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<odoo>
+    <!-- Action to be able to display the model (as a tree, form view) -->
+    <record id="estate_property_model_action" model="ir.actions.act_window">
+        <field name="name">Properties</field>
+        <field name="res_model">estate.property</field>
+        <field name="view_mode">tree,form</field>
+        <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">
+                Access to the properties
+            </p>
+        </field>
+    </record>
+
+    <!-- Tree view -->
+    <record id="estate_property_view_tree" model="ir.ui.view">
+        <field name="name">estate.property.tree</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <tree string="Channel">
+                <field name="name"/>
+                <field name="postcode"/>
+                <field name="bedrooms"/>
+                <field name="living_area"/>
+                <field name="expected_price"/>
+                <field name="selling_price"/>
+                <field name="date_availability"/>
+            </tree>
+        </field>
+    </record>
+
+    <!-- Form view -->
+    <record id="estate_property_view_form" model="ir.ui.view">
+        <field name="name">estate.property.form</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <form string="Channel">
+                <sheet>
+                    <widget name="web_ribbon" title="To sold" bg_color="bg-success"/>
```

Comment:
What's the use of the widget exactly ?

Reply:
I just wanted to try a display for available properties, but i can remove it if inappropriate for the tutorial